src/backend/distributed/Makefile

```diff
@@ -97,7 +97,7 @@ $(EXTENSION)--6.0-18.sql: $(EXTENSION)--6.0-17.sql $(EXTENSION)--6.0-17--6.0-18.
 
 NO_PGXS = 1
 
-SHLIB_LINK = $(libpq)
+SHLIB_LINK += $(libpq)
```

Comment:
(I figured out why, but this kind of thing might require a comment in the commit message next time)